### DISCUSSION TITLE
Artic code vault contributor badge

### DISCUSSION THREAD

**Discussion by EJL3** [github/india#101] [Open Source] [answered]
The last archive program was held in feb 2020. so for the users who missed the badge will there be a archive snapshot soon?

**Comment by DavideC03** [comment, marked as answer]
As you can read in the [FAQ](https://archiveprogram.github.com/faq/): 
```We plan to evaluate the program, and the state of the art of archival technology, every five years. Depending on the results of each evaluation, we may then decide to take another snapshot of GitHub’s public code and archive it in cold storage.```

**Reply by mishmanners** [reply]
And answered here too: https://github.com/github/india/discussions/97

**Reply by EJL3** [reply]
Thank you man @DavideC03
